### PR TITLE
app-i18n/mozc: add fcitx5 support

### PR DESCRIPTION
**Pull request by liangyongxiang**
Bug: https://bugs.gentoo.org/760501

### CODE REVIEW

File: app-i18n/mozc/mozc-2.28.5029.102-r2.ebuild [hunk 1]
```diff
@@ -130,11 +144,14 @@ src_unpack() {
 	if [[ "${PV}" == "9999" ]]; then
 		git-r3_src_unpack
 
-		if use fcitx4; then
+		if use fcitx4 || fcitx5; then
```
Comment (liuyujielol) [inline app-i18n/mozc/mozc-2.28.5029.102-r2.ebuild:147]:
```suggestion
		if use fcitx4 || use fcitx5; then
```

**Comment by liangyongxiang** [comment]
Refresh the year of copyright.

**Comment by a17r** [comment]
@liangyongxiang: This revbump dropped ~arm64 keyword - why exactly? (~ppc64 is clear to me from looking at revdeps)

This leaves a messy situation for cleanups:
```
Keywords for app-i18n/mozc:
                 |                             |   u   |  
                 | a   a     p s   a l   r     |   n   |  
                 | m   r h   p p   l o m i s m | e u s | r
                 | d a m p p c a x p o i s 3 6 | a s l | e
                 | 6 r 6 p p 6 r 8 h n p c 9 8 | p e o | p
                 | 4 m 4 a c 4 c 6 a g s v 0 k | i d t | o
-----------------+-----------------------------+-------+-------
2.28.5029.102    | + o ~ o o + o + o o o o o o | 8 o 0 | gentoo
2.28.5029.102-r1 | ~ o ~ o o ~ o ~ o o o o o o | 8 o   | gentoo
2.28.5029.102-r4 | ~ o o o o o o ~ o ~ o o o o | 8 o   | gentoo
```

**Comment by liangyongxiang** [comment]
> @liangyongxiang: This revbump dropped ~arm64 keyword - why exactly? (~ppc64 is clear to me from looking at revdeps)
> 
> This leaves a messy situation for cleanups:
> 
> ```
> Keywords for app-i18n/mozc:
>                  |                             |   u   |  
>                  | a   a     p s   a l   r     |   n   |  
>                  | m   r h   p p   l o m i s m | e u s | r
>                  | d a m p p c a x p o i s 3 6 | a s l | e
>                  | 6 r 6 p p 6 r 8 h n p c 9 8 | p e o | p
>                  | 4 m 4 a c 4 c 6 a g s v 0 k | i d t | o
> -----------------+-----------------------------+-------+-------
> 2.28.5029.102    | + o ~ o o + o + o o o o o o | 8 o 0 | gentoo
> 2.28.5029.102-r1 | ~ o ~ o o ~ o ~ o o o o o o | 8 o   | gentoo
> 2.28.5029.102-r4 | ~ o o o o o o ~ o ~ o o o o | 8 o   | gentoo
> ```

For ~arm64: At that time app-i18n/fcitx:5 didn't have the arm64 keyword yet.
For ~ppc and ~pcc64: I was still requesting the keyword app-i18n/fcitx:5, https://bugs.gentoo.org/945543

I think it's in https://bugs.gentoo.org/922103 I missed the re-keyword for mozc.

**Comment by a17r** [comment]
Thanks for the references, amended my bug.

**Pull request closed without merging**